server/views/pages/induction/prePrisonEducation/qualificationLevel.njk

```diff
@@ -19,111 +19,119 @@ Data supplied to this template:
 {% endblock %}
 
 {% block content %}
-  <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
+  <form class="form" method="post" novalidate="">
+    <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
 
-      <form class="form" method="post" novalidate="">
-        <div class="govuk-form-group">
-          <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
-        
-          {{ govukRadios({
-            name: "qualificationLevel",
-            fieldset: {
-              legend: {
-                text: title,
-                isPageHeading: true,
-                classes: "govuk-fieldset__legend--l"
-              }
-            },
-            hint: {
-              text: 'Add their highest level of qualification first. You can add any others afterwards.'
+    <div class="govuk-grid-row">
```

Comment:
Because of the required layout, where the sidebar is horizontally in line with the top of the first radio button, I've had to move the form legend (page title) and hint text out of the `govukRadios` `fieldset` and essentially hardcode the markup for them here in their own row.
And in doing that I've had to move the `form` element to wrap the whole lot to make it semantically correct 👍